src/sections/collection-list.liquid

```diff
@@ -7,10 +7,13 @@
     {%- assign collection = collections[block.settings.collection] -%}
 
     <a href="{% if collection == empty %}#{% else %}{{ collection.url }}{% endif %}">
-      {% if collection.image != blank %}
-        {{ collection | img_url: '480x480' | img_tag: collection.title }}
-      {% elsif collection.products.first != blank %}
-        {{ collection.products.first | img_url: '480x480' | img_tag: collection.title }}
+      {% if collection.image != blank or collection.products.first != blank %}
+        {%- assign image_objet = collection.image | default: collection.products.first.featured_image -%}
+        {% include 'responsive-image' with
+          image: image_objet,
+          width: 480,
```

Comment:
Maybe we should call these `max-width` and `max-height`?

Reply:
@t-kelly I renamed to `max_width` and `max_height` everywhere when we do `include`. In the snippet code, since there were already variables named `max_width` and `max_height`, I changed into `max_height_image` and `max_height_image`